families/identity/sawtooth_identity/Cargo.toml

```diff
@@ -0,0 +1,59 @@
+#
+#  Copyright 2018 Bitwise IO, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+[package]
+name = "sawtooth_identity"
```

Comment:
The binary produced should be named `identity-tp` to be consistent with the naming scheme for other core tps and to match the previous entry point. See https://github.com/hyperledger/sawtooth-core/blob/master/families/settings/sawtooth_settings/Cargo.toml#L38 for an example of how to do this.